express/src/lib/errors.ts

```diff
@@ -2,35 +2,3 @@ export interface RenderOptions {
     values?: {[name: string]: string};
     errorMessages?: {[name: string]: string};
 }
-
```

Comment:
It should be possible to delete this whole file.  You can inline the RenderOptions type in this only place it's used with something like:

```
export default function emailIsPresentInSession(
    template: string,
    renderOptions: {
        values?: {[name: string]: string};
        errorMessages?: {[name: string]: string};
    }
):
```
There's a good chance that middleware will be changed again at some point in the future so I think it's okay to put that type there for now.

Reply:
Sure. Done.